_episodes/02-getting-started.md

```diff
@@ -1,29 +1,117 @@
 ---
-title: "Getting started with git"
-teaching: 20
+title: "Getting started with Git"
+teaching: 25
 exercises: 0
 questions:
 - "What are repositories and how are they created?"
 - "What do `add` and `commit` mean?"
 - "How do I check the status of my repository?"
 objectives:
-- "create a git repository"
-- "track changes to files using the git repository"
-- "query the current status of the git repository"
+- "create a Git repository"
+- "track changes to files using the Git repository"
+- "query the current status of the Git repository"
 keypoints:
 - "When you initialize a Git repository in a directory, Git starts tracking the changes you make inside that directory."
 - "This tracking creates a history of the way the files have changed over time."
 - "Git uses a two-step process to record changes to your files. Changes to files must first be added to the staging area, then committed to the Git repository."
 ---
 
-### Using Git
+### Setting up Git
 
-One of the main barriers to getting started with git is the language. Although some of the language used in git is
-fairly self-explanatory, other terms are not so clear. The best way to get to learn the language - which consists of a
-number of verbs such as `add`, `commit` and `push` (preceded by the word 'git') - is by using it, which is what we will be doing during this
-lesson. These commands will be explained as we proceed from setting up a new version-controlled project to publishing
-our own website.
+When we use Git on a new computer for the first time,
+we need to configure a few things. The basic elements of a configuration for Git are:
+
+*   your name and email address,
+*   what your preferred text editor is,
+*   set the name of your default branch (branches are an important component of Git that we will cover later)
+*   and that you want to use these settings globally (i.e. for every project).
+
+First, we will tell Git our user name and email. For this lesson, we will be interacting with [GitHub](https://github.com/) and so we want to use the same email address we used when we set up our GitHub account. If you are concerned about privacy, please review [GitHub's instructions for keeping your email address private][git-privacy].
```

Comment:
On Line 29: 
the format is 
\[GitHub’s instructions for keeping your email address private]\(https://help.github.com/articles/keeping-your-email-address-private/)

Reply:
Thank you so much -- excellent catch. I revised and did a new commit.

Reply:
For the sake of completeness, and because you may very well see both forms used elsewhere in Carpentries lessons (sometimes together in the same page/lesson), the 

```markdown
[GitHub's instructions for keeping your email address private][git-privacy]
```

syntax is also valid, if you include a link reference later in the file, e.g. 

```markdown
[GitHub's instructions for keeping your email address private][git-privacy]

[git-privacy]: https://help.github.com/articles/keeping-your-email-address-private/
```

It is standard practice to include all such link references at the very bottom of the Markdown file. These "reference-style" links in Markdown are particularly useful for URLs that are linked to in multiple locations throughout the document: that way, if the address of the target page changes, you only need to modify the URL in a single location to update all the relevant links.

To be clear, I am not suggesting that you need to take this approach here, but adding the information because it may be helpful when reading/contributing to other repositories.

Reply:
Thank you @tobyhodges! I remember being so confused by this when I started working with Carpentries lessons.

Reply:
I think I will come back and improve the linking syntax and adding the link references to the bottom of the Markdown file in a separate pull request. I need to learn this standard practice, but it looks like what I used is functional for the time being.

Reply:
Absolutely! Indeed, I meant to say that, if using the reference-style syntax for links, then it is standard practice to put the references at the end of the file. But plenty of projects, including many Carpentries lessons, use the "inline" syntax you are using here. I agree it is a minor thing and best left to a separate PR, if you want to do it at all.